### PR TITLE
(MODULES-8091) Prep module for 2.1.0 release

### PR DESCRIPTION
**Pull request by glennsarti**
The commit prepares the module for the 2.1.0 release.

### CODE REVIEW

**Comment by MikaelSmith** [comment]
This needs an update based on https://github.com/puppetlabs/puppetlabs-reboot/pull/178.

**Comment by glennsarti** [comment]
I'll wait for 179 and 176 to land unless urgent.

**Comment by MikaelSmith** [comment]
It sounded like there might be other release drivers. I definitely don't need to wait on the other work.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+## [2.1.0] - 2018-12-06
+### Added
+- Use wait_until_available to reduce task runs ([BOLT-956](https://tickets.puppetlabs.com/browse/BOLT-956))
+- Add bash and powershell implementations of reboot task ([BOLT-459](https://tickets.puppetlabs.com/browse/BOLT-459))
+- Add wait function for Puppet Plans ([MODULES-8046](https://tickets.puppetlabs.com/browse/MODULES-8046))
```
Comment (MikaelSmith) [inline CHANGELOG.md]: This was reverted. Should we still include it? Is it always going to be re-added by automatic changelog generation?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+## [2.1.0] - 2018-12-06
+### Added
+- Use wait_until_available to reduce task runs ([BOLT-956](https://tickets.puppetlabs.com/browse/BOLT-956))
```
Comment (MikaelSmith) [inline CHANGELOG.md]: This ticket number was accidental, it no longer exists :(

Comment (glennsarti) [reply CHANGELOG.md]: 956 exists.  The commit says 957

File: CHANGELOG.md (hunk 1 above)
Comment (glennsarti) [reply CHANGELOG.md]: Whoops yes.  I missed the revert.  Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (MikaelSmith) [reply CHANGELOG.md]: Oh ok. Close enough.

**Pull request merged**